molecule/_resources/prepare.yml

```diff
@@ -0,0 +1,58 @@
+---
+
+- name: Prepare hosts
+  hosts: all
+  tasks:
+    # to ensure having a real FQDN
+    # (dots are not allowed in LXD container names, thus sticking to short names)
+    - name: set proper hostname
+      hostname:
+        name: "{{ ansible_nodename }}.example.com"
+      # using the `hostname` instantiation parameter for Docker, as /etc/hostname cannot be modified
+      when: ansible_virtualization_type != 'docker'
+    - name: Install needed tools (EL)
+      package:
+        name: "{{ item }}"
+        state: installed
+      with_items:
+        - epel-release
+      when: ansible_distribution == 'CentOS' or ansible_distribution == 'RedHat'
+
+- name: Prepare builder host
+  hosts: ansible-test-builder
+  tasks:
+    - name: Install builder tools
+      package:
+        name: "{{ item }}"
+        state: installed
+      with_items:
+        - sudo              # needed for tests
+        - openssh-clients   # to communicate between machines
+        - crontabs          # to setup cron jobs
+
+- name: Prepare web host
+  hosts: ansible-test-web
+  tasks:
+    - name: Install builder tools
+      package:
+        name: "{{ item }}"
+        state: installed
+      with_items:
```

Comment:
no need to use a loop with one item.

Reply:
right. In fact `package` does not need loops anymore and that's the recommended usage, so I will fix.